### PR TITLE
Overwrite bazel if /usr/bin/bazel already exists.

### PR DESCRIPTION
**Pull request by ailzhang**

### CODE REVIEW

**Review by seemethere — APPROVED** [review]

File: .jenkins/pytorch/build.sh [hunk 1]
```diff
@@ -259,7 +259,7 @@ if [[ "${BUILD_ENVIRONMENT}" == *xla* ]]; then
   # XLA build requires Bazel
   # We use bazelisk to avoid updating Bazel version manually.
   sudo npm install -g @bazel/bazelisk
-  sudo ln -s "$(command -v bazelisk)" /usr/bin/bazel
```
Comment (malfet) [inline .jenkins/pytorch/build.sh:262]: Why not call `get_bazel` function from `.jenkins/pytorch/common.sh`?

Comment (ailzhang) [reply .jenkins/pytorch/build.sh:262]: IMHO `bazelisk` is better than our `get_bazel` function, which installs a fixed version of bazel. `bazelisk` is able to read a `bazelversion` file in `tensorflow` and pull in the right version automatically. 
It saves us from manually updating bazel version in pytorch CI.

Comment (ailzhang) [reply .jenkins/pytorch/build.sh:262]: I think we should similarly add a `.bazelversion` file in pytorch and start using bazelisk in CI. 
It has the benefit that:
- Easier to manage and update
- User would know which bazelversion to work with.

**Comment by gchanan** [comment]
I'll merge this and we can work out a better solution later.

**Pull request merged**